scripts/sct_download_data.py

```diff
@@ -127,24 +128,46 @@ def main(args=None):
     data_name = arguments['-d']
     verbose = int(arguments.get('-v'))
     sct.init_sct(log_level=verbose, update=True)  # Update log level
-    dest_folder = arguments.get('-o', os.path.abspath(os.curdir))
+    dest_tmp_folder = arguments.get('-o', os.path.abspath(os.curdir)+'/tmp')
```

Comment:
"/" doesn't work in Windows, and what if user already has a local folder called "./tmp/"?
there are better ways to generate temp folders, see class `sct_utils.TempFolder`

Reply:
Oh ok, nice one, looking at it.